src/main/java/seedu/address/logic/commands/DeletePersonCommand.java

```diff
@@ -13,23 +13,26 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using its displayed index from the contact list of the application.
  */
 public class DeletePersonCommand extends Command {
 
     public static final String COMMAND_WORD = "deletePerson";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the person based on the index number in the contact list displayed.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
 
-    private final Index targetIndex;
-
-    public DeletePersonCommand(Index targetIndex) {
-        this.targetIndex = targetIndex;
+    private final Index index;
+    /**
+     * Creates a DeletePersonCommand.
+     * @param index displayed index of Event to be deleted.
```

Comment:
```suggestion
     * @param index displayed index of Person to be deleted.
```
Might be a copy-over error